src/tinyformat.h

```diff
@@ -1063,6 +1063,6 @@ std::string format(const std::string &fmt, const Args&... args)
 
 } // namespace tinyformat
 
-#define strprintf tfm::format
+#define strprintf tinyformat::format
```

Comment:
Please submit those changes upstream as well

Reply:
The modified line is not part of the upstream tinyformat.h but of the Bitcoin Core additions (see comment in line 1055)